src/code/dialogs/importDialog.js

```diff
@@ -70,7 +70,7 @@ const ImportDialog = WDialog.extend({
   },
 
   drawToolsFormat() {
-    const dtitems = localStorage["plugin-draw-tools-layer"];
+    const dtitems = localStorage["window.plugin.drawTools.KEY_STORAGE"];
```

Comment:
remove the quotes, it is not a static string, it is a variable.

Reply:
Updated